docs/verbs/catkin_build.rst

```diff
@@ -480,8 +480,20 @@ To run catkin tests for catkin packages, use the following:
 
 .. code-block:: bash
 
+    $ catkin run_tests --verbose -i
```

Comment:
`--verbose` is redundant, as it is already in the `run_test` alias, which expands like so:

```
==> Expanding alias 'run_tests' from 'catkin run_tests' to 'catkin build --verbose --catkin-make-args run_tests --'
```

Also, does recommending `-i` make sense? It would jumble up the output.